ui/app/models/config-ui/message.js

```diff
@@ -22,6 +22,28 @@ const validations = {
       message: 'Link title and url are required.',
     },
   ],
+  startTime: [
+    {
+      validator(model) {
+        if (!model.endTime) return true;
+        const start = new Date(model.startTime);
+        const end = new Date(model.endTime);
+        return isBefore(start, end);
```

Comment:
If the start and end times are ISO strings you can pass those directly to `isBefore` without needing to create date objects.

Reply:
These strings are in datetime local format. :)

Reply:
Sounds good. `isBefore` should accept that format without needing to convert to a date object but just more of an FYI than anything 😄

Reply:
Good to know I'll update that!